src/hardware/mixer.cpp

```diff
@@ -24,8 +24,12 @@
 
 // #define DEBUG 1
 
+#include "mixer.h"
```

Comment:
Suggestion: empty line after main header include (that's based on [Google Style](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes), which is mentioned in CONTRIBUTING.md).

Reply:
Will do. I've been reluctant to run the reformatter later for fear of it messing up a bunch of code (and changing things in adjacent lines).  I will resume using it and just be careful about what I accept.